playbook-website/app/components/lg_hp_social_proof_component.erb

```diff
@@ -0,0 +1,114 @@
+<%= pb_rails("background", props: {
+    display: { xs: "none", sm: "none", md: "none", lg: "flex", xl: "flex"  },
+    classname: "middle-white-background",
+    background_color: "white",
+    padding_top: "xl",
+    padding_bottom: "xl"
+}) do %>
+    <%= pb_rails("flex", props: {
+        classname: "hp-social-proof-background-width",
+        padding_left: "md",
+        padding_right: "md",
+        max_width: "xl",
+        justify: "center",
+        wrap: true
+    }) do %>
+        <%= pb_rails("background", props: {
+            classname: "hp-social-proof-background-width",
+            background_color: "dark",
+            border_radius: "lg"
+        }) do %>
+            <%= pb_rails("background", props: {
+                background_color: "dark",
+                image_url: asset_pack_path("media/images/hp-social-proof-background.png"),
+                border_radius: "lg",
+                background_repeat: "no-repeat",
+                background_position: "center right",
+                background_size: "auto",
+                alt: "background",
+                transition: "fade",
+                padding: "xl"
+            }) do %>
+
+                <%= pb_rails("flex", props: {
+                    horizontal: "center",
```

Comment:
hey @Israel-Molestina , horizontal as a prop on the flex kit is deprecated, can we change this to the replacement? I think it's justify?